src/main/java/org/alliancegenome/curation_api/model/entities/AlleleDiseaseAnnotation.java

```diff
@@ -19,11 +28,12 @@
 @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
 @Schema(name = "Allele_Disease_Annotation", description = "Annotation class representing a allele disease annotation")
 @JsonTypeName("AlleleDiseaseAnnotation")
+@OnDelete(action = OnDeleteAction.CASCADE)
 public class AlleleDiseaseAnnotation extends DiseaseAnnotation {
 
     @IndexedEmbedded(includeDepth = 1)
     @IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
-    @ManyToOne(cascade = {})
+    @ManyToOne
     @org.hibernate.annotations.OnDelete(action = org.hibernate.annotations.OnDeleteAction.CASCADE)
```

Comment:
Ahh ok with the class level @OnDelete do we need both of theses?

Reply:
My understanding was that the class-level annotation was for cascading the delete of diseaseannotations with gene/allele/agmdiseaseannotations, whereas this is to cascade the deletion of referenced genes.

Reply:
ok